ros2bag/ros2bag/verb/info.py

```diff
@@ -50,8 +54,11 @@ def main(self, *, args):  # noqa: D102
         if args.topic_name and args.verbose:
             print("Warning! You have set both the '-t' and '-v' parameters. The '-t' parameter "
                   'will be ignored.')
+        if not args.verbose and args.size_contribution:
+            print("Warning! You have set the '--size-contribution' parameter without the '-v' parameter. "
+                  "The '--size-contribution' parameter will be ignored.")
         if args.verbose:
-            Info().read_metadata_and_output_service_verbose(args.bag_path, args.storage)
+            Info().read_metadata_and_output_service_verbose(args.bag_path, args.storage, args.size_contribution)
```

Comment:
I don't think that we need extra parameter like `--size-contribution` in addition to the `--verbose` argument.
Let's just add size calculation by default to the `--verbose` CLI option.